src/System.Diagnostics.Process/src/System/Diagnostics/Process.Win32.cs

```diff
@@ -28,6 +30,8 @@ private bool StartCore(ProcessStartInfo startInfo)
 
         private unsafe bool StartWithShellExecuteEx(ProcessStartInfo startInfo)
         {
+            string arguments;
```

Comment:
We generally prefer that local variables be declared as close to their point of initialization as possible.